packages/cgb-scripts/template/src/init.php

```diff
@@ -28,38 +30,36 @@ function <% blockNamePHPLower %>_cgb_block_assets() {
 		array( 'wp-blocks' ) // Dependency to include the CSS after it.
 		// filemtime( plugin_dir_path( __DIR__ ) . 'dist/blocks.style.build.css' ) // Version: filemtime — Gets file modification time.
 	);
-} // End function <% blockNamePHPLower %>_cgb_block_assets().
-
-// Hook: Frontend assets.
-add_action( 'enqueue_block_assets', '<% blockNamePHPLower %>_cgb_block_assets' );
 
-/**
- * Enqueue Gutenberg block assets for backend editor.
- *
- * `wp-blocks`: includes block type registration and related functions.
- * `wp-element`: includes the WordPress Element abstraction for describing the structure of your blocks.
- * `wp-i18n`: To internationalize the block's text.
- *
- * @since 1.0.0
- */
-function <% blockNamePHPLower %>_cgb_editor_assets() {
-	// Scripts.
+	// Block scripts.
 	wp_enqueue_script(
```

Comment:
Did you test this? Coz. it's wrong. You're enqueuing the files and then register them. Which is wrong. I hope I get time to write automated tests for this but till then do your own due diligence.

Reply:
You're right about the enqueuing. This is a mistake from my side. Although when I ran the tests, I was registering the scripts first and everything was working fine. Going to correct the PR.

Here is the file of the block that I tested:

```php
<?php
/**
 * Blocks Initializer
 *
 * Enqueue CSS/JS of all the blocks.
 *
 * @since   1.0.0
 * @package CGB
 */

// Exit if accessed directly.
if ( ! defined( 'ABSPATH' ) ) {
	exit;
}

/**
 * Enqueue Gutenberg block assets for both frontend + backend.
 *
 * `wp-blocks`: includes block type registration and related functions.
 * `wp-element`: includes the WordPress Element abstraction for describing the structure of your blocks.
 * `wp-i18n`: To internationalize the block's text.
 *
 * @since 1.0.0
 */
function my_block_cgb_block_assets() {
	// Styles.
	wp_register_style(
		'my_block-cgb-style-css', // Handle.
		plugins_url( 'dist/blocks.style.build.css', dirname( __FILE__ ) ), // Block style CSS.
		array( 'wp-blocks' ) // Dependency to include the CSS after it.
		// filemtime( plugin_dir_path( __DIR__ ) . 'dist/blocks.style.build.css' ) // Version: filemtime — Gets file modification time.
	);

	// Block scripts.
	wp_register_script(
		'my_block-cgb-block-js', // Handle.
		plugins_url( '/dist/blocks.build.js', dirname( __FILE__ ) ), // Block.build.js: We register the block here. Built with Webpack.
		array( 'wp-blocks', 'wp-i18n', 'wp-element' ), // Dependencies, defined above.
		false, // filemtime( plugin_dir_path( __DIR__ ) . 'dist/blocks.build.js' ), // Version: filemtime — Gets file modification time.
		true // Enqueue the script in the footer.
	);

	// Block backend styles.
	wp_register_style(
		'my_block-cgb-block-editor-css', // Handle.
		plugins_url( 'dist/blocks.editor.build.css', dirname( __FILE__ ) ), // Block editor CSS.
		array( 'wp-edit-blocks' ) // Dependency to include the CSS after it.
		// filemtime( plugin_dir_path( __DIR__ ) . 'dist/blocks.editor.build.css' ) // Version: filemtime — Gets file modification time.
	);

	/**
	 * Register the block scripts and styles for both frontend and
	 * backend to ensure they are enqueued when the editor loads.
	 *
	 * @link https://wordpress.org/gutenberg/handbook/blocks/writing-your-first-block-type/
	 */
	register_block_type( 'cgb/block-my-block', array(
		'style' => 'my_block-cgb-style-css', // Enqueue blocks.style.build.css on both frontend & backend.
		'editor_script' => 'my_block-cgb-block-js', // Enqueue blocks.build.js in the editor only.
		'editor_style' => 'my_block-cgb-block-editor-css', // Enqueue blocks.editor.build.css in the editor only.
	) );
} // End function my_block_cgb_block_assets().

// Hook: Block assets.
add_action( 'init', 'my_block_cgb_block_assets' );

```

Reply:
Corrected this mistake.